pkg/roachpb/metadata.go

```diff
@@ -946,7 +956,53 @@ func (h *GCHint) ForwardLatestRangeDeleteTimestamp(ts hlc.Timestamp) bool {
 	return false
 }
 
-// ResetLatestRangeDeleteTimestamp resets delete range timestamp.
-func (h *GCHint) ResetLatestRangeDeleteTimestamp() {
-	h.LatestRangeDeleteTimestamp = hlc.Timestamp{}
+// ScheduleGCFor updates the hint to schedule GC for the given timestamp.
```

Comment:
```suggestion
// ScheduleGCFor updates the hint to schedule eager GC for data up to the given
// timestamp. When this timestamp falls below the GC threshold/TTL, it will
// be eagerly enqueued for GC when considered by the MVCC GC queue.
```